examples/python/comparing_simulators.py

```diff
@@ -253,9 +254,9 @@
 
 # Run on the backend:
 
-backend = QsharpEstimatorBackend()
-c = backend.get_compiled_circuit(c)
 # (disabled because of https://github.com/CQCL/pytket-qsharp/issues/37)
+# backend = QsharpEstimatorBackend()
```

Comment:
Just to check....

We can still have some qsharp content in the examples but we cannot run circuits on some qsharp backends?

Reply:
The only part of pytket-qsharp that is currently not working is the `QsharpEstimatorBackend`, more details about this can be found in https://github.com/CQCL/pytket-qsharp/issues/37.
All other parts of pytket-qsharp should be working fine.